src/org/scada_lts/dao/GenericDaoCR.java

```diff
@@ -32,7 +32,7 @@
 	
 	static final String LIMIT = " LIMIT ";
 	static final int NO_LIMIT = 0;
-	
+	//List<T> findAllW();
```

Comment:
To delete

Reply:
I do not know why you see this. It has been deleted yesterday and pushed.